content/blog/2022-07-05-serving-models-with-mlem.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Serving Machine Learning Models with MLEM
+date: 2022-07-05
+description: >
+  Once you have a machine learning model that's ready for production, getting it
+  out can be complicated. In this tutorial, we're going to use MLEM to deploy a
+  model to an API.
+descriptionLong: >
+  Getting a machine learning model out to our end users can be an involved task.
+  You have to make sure it's in the correct format for the environment and for
+  the way it will be consumed. We're going to walk through how you can use MLEM
+  to save your model and serve it with a web API.
+picture: 2022-07-05/serve_mlem_models.png
+pictureComment: Serving Machine Models with MLEM
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/syncing-data-to-aws-s3/1192
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - MLEM
+  - Collaboration
+---
+
+Training a machine learning model is only one step in the process of getting
+something useful out to end-users. There are many goals for machine learning
+projects and we need accurate models that provide us with information based on
+the data received from a production environment. That could mean data coming
+from an API or batch processing, for example.
+
+Either way, you'll need to save your trained and validated model in a format
+that's consumable by other systems. That's why we'll be covering how to serve
+models through a [REST](https://restfulapi.net/) endpoint or a Python package
+with [MLEM](https://mlem.ai/).
+
+> You can get the repo we're working with
+> [here](https://github.com/iterative/stale-model-example/tree/mlem-serve).
+
+## Take a candidate model
+
+There are instructions in the project
+[README](https://github.com/iterative/stale-model-example/tree/mlem-serve#readme)
+on how to get everything you need installed and running. This is a simple ML
+project that uses DVC for data versioning and experiment tracking.
```

Comment:
```suggestion
on how to get everything you need installed and running. This is a simple ML
project that uses [DVC](https://dvc.org/) for data versioning and experiment
tracking.
```

^ I understand the blog will be extracted from dvc.org in the near future so something to keep in mind.

p.s. consider linking "data versioning" and/or "experiment tracking" to specific docs too if it doesn't feel too crowded. Or combine into "[experiment versioning](https://dvc.org/doc/user-guide/experiment-management/experiments-overview)" ™️ .